### PR TITLE
removed/commented search bar

### LINKED ISSUE #50: [Bug] Remove Search Documentation Bar
### Description

We don't need Search Bar right now.

image

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Huzaifa785**
## Fixes Issue

Fixed #50 

## Changes proposed

- [ ✅] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ✅] This PR does not contain plagiarized content.
- [ ✅] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-10-01 at 2 22 19 PM

## Note to reviewers

I commented the search bar code as of now, so if we need it in the future, we may uncomment it and start using it.

### CODE REVIEW

File: content/batch/learn/css/grid.mdx [hunk 1]
```diff
@@ -36,6 +36,143 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
    We hope that you found the tutorial video helpful in understanding the basic concepts of CSS Grid.
 </Callout>
 
+## Notes
```
Comment (ManishBisht777) [inline content/batch/learn/css/grid.mdx:39]: Why these changes are here?
make sure to limited the pr for only one issue at a time from next time

Comment (Huzaifa785) [reply content/batch/learn/css/grid.mdx:39]: Oh okay. So sorry for it, will keep in mind from next time

Comment (Huzaifa785) [reply content/batch/learn/css/grid.mdx]: Hi

I removed that code grid docs code, are there any more changes to be done?

Comment (ManishBisht777) [reply content/batch/learn/css/grid.mdx]: Thanks for the quick changes. LGTM

**Review by ManishBisht777 — APPROVED** [review]
